unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -49,6 +49,8 @@
     5, 5, 5, 5,
     6, 6, 6, 6};
 
+  public static UTF8String EMPTY_STRING = new UTF8String(new byte[0], BYTE_ARRAY_OFFSET, 0);
```

Comment:
might be more clear to call this EMPTY_UTF8 to avoid confusion with java String.

Also can you use the fromString constructor? Performance won't matter too much here since it is a static one.